src/lib/components/layout/v2/index.tsx

```diff
@@ -2,12 +2,12 @@ import { ReactElement } from 'react';
 import styled from 'styled-components';
 import { navbarHeight } from '../../../style/theme';
 
-const LayoutContainer = styled.div`
+const LayoutContainer = styled.div<{ variant?: 'transparent' }>`
   display: flex;
   flex-direction: column;
   height: 100vh;
   box-sizing: border-box;
-  background: ${(props) => props.theme['backgroundLevel1']};
+  background: ${props => props.variant === 'transparent' ? 'transparent' : props.theme.backgroundLevel1};
```

Comment:
I wonder if the variant shouldn't be defined on the App container instead. The reason motivating this thinking is that the layout component in our apps is often defined within shell-ui, whereas it is up to the micro app to decide to have a glow background due to its responsibility.